### PR TITLE
v0.10 CHANGELOG-0.10.md update

### PR DESCRIPTION
**Pull request by rafzei**

### CODE REVIEW

File: CHANGELOG-0.10.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog 0.10
 
+Note: Due to recent changes in Epiphany source code, Support Policy and to keep Semantic Versioning standards we decided to increase Epiphany Major version.
+Version 0.10 won't be supported anymore. Instead, we introduced version 1.0 which includes all changes from version 0.10.
+
```
Comment (seriva) [inline CHANGELOG-0.10.md:5]: **bold** maybe?

**Review by seriva — COMMENTED** [review]
Or I will just use this comment to remind you to add this change to develop as well;)

File: CHANGELOG-0.10.md (hunk 1 above)
Comment (rafzei) [reply CHANGELOG-0.10.md:5]: sure, let's do this bolded

**Comment by rafzei** [comment]
> Or I will just use this comment to remind you to add this change to develop as well;)

PR is waiting for you **but** I have to add bold :cat:

File: CHANGELOG-0.10.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changelog 0.10
 
+**Note: Due to recent changes in Epiphany source code, Support Policy and to keep Semantic Versioning standards we decided to increase Epiphany Major version.
+Version 0.10 won't be supported anymore. Instead, we introduced version 1.0 which includes all changes from version 0.10.**
+
```
Comment (mkyc) [inline CHANGELOG-0.10.md:5]:
```suggestion

❗ ❗ ❗ 🎉  Please switch to v1.0.x 🎉 ❗ ❗ ❗ 

```

**Review by mkyc — CHANGES_REQUESTED** [review]

**Review by seriva — APPROVED** [review]

**Review by mkyc — APPROVED** [review]

**Pull request merged**